packages/calendar/src/components/input-types/date-range-dual-text-input/DateRangeDualTextInput.tsx

```diff
@@ -169,6 +171,7 @@ export function DateRangeDualTextInput<TData>({
         <CalendarWithMonthSwitcher
           onClickDay={onClickDay}
           dateInFocus={dateInFocus}
+          firstMonth={useDateInFocusForCalendarMonth ? dateInFocus : undefined}
```

Comment:
Maybe we should expose `firstMonth` instead?

Reply:
Yes, I think that is a better option.